tensorflow/lite/kernels/activations_test.cc

```diff
@@ -108,10 +108,20 @@ class BaseActivationsOpModel : public SingleOpModel {
   BaseActivationsOpModel(TensorData input, float alpha) {
     input_ = AddInput(input);
     // The output scale and input scale might be different.
-    if (input.type == TensorType_UINT8 || input.type == TensorType_INT8) {
+    if (input.type == TensorType_UINT8 || input.type == TensorType_INT8 ||
+        input.type == TensorType_INT16) {
       auto output_min = (input.min >= 0) ? input.min : input.min * alpha;
       auto output_max = (input.max >= 0) ? input.max : input.max * alpha;
-      output_ = AddOutput({input.type, {}, output_min, output_max});
+      if (input.type == TensorType_INT16) {
+        output_ = AddOutput({TensorType_INT16,
+                           {},
+                           output_min,
+                           output_max,
+                           1.0f / (std::numeric_limits<int16_t>::max() + 1),
```

Comment:
is here trying to set the scale?

Reply:
Hi @renjie-liu ! Thanks for pointing this out.  I corrected.